### DISCUSSION TITLE
Include styling guide in documentation

### DISCUSSION THREAD

**Discussion by promethyttrium** [chakra-ui/ark#3072] [Q&A]
The various examples for a component like the Switch, it does not even come with basic styles so we have an idea where to start to style it. At least give us something to start with. I know the suggestion is to use Park UI if someone wants pre-styled components but I feel like that misses the mark. I want to use Ark-UI because I want a blank slate -- I just want to know what I can do with it without having to dive into the code.

**Comment by cschroeter** [comment]
@promethyttrium 

Hey mate, good point! Documentation is like a painting—you can always add more details and guidance. Basic styling for components is one of those areas.

To be honest, it’s not our priority right now. However, you can check out our Storybook styles (vanilla CSS) as a starting point:
https://github.com/chakra-ui/ark/tree/main/packages/react/.storybook/styles

**Comment by segunadebayo** [comment]
This now exists in the docs.
